core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -121,7 +129,7 @@
   private TCredentials rpcCreds;
   private ThriftTransportPool thriftTransportPool;
 
-  private volatile boolean closed = false;
```

Comment:
Why was volatile removed?  W/o volatile or synchronization, changes may not be seen.  Like `ensureOpen()` may not see changes.

Reply:
I think it was lost in a prior change, which I then reverted. I will add it back.

Reply:
Addressed in 47e90dca559f02683c7c9f58ae366a422367a2d7